src/c++/perf_analyzer/test_perf_utils.cc

```diff
@@ -114,4 +117,288 @@ TEST_CASE("testing the ParseTensorFormat function")
   CHECK(ParseTensorFormat("") == cb::TensorFormat::UNKNOWN);
 }
 
+TEST_CASE("perf_utils: ParseProtocol")
```

Comment:
Since you've added this here, can you remove it from `src/c++/perf_analyzer/perf_utils.cc`?